api/src/main/java/io/grpc/ForwardingChannelBuilder.java

```diff
@@ -91,6 +91,12 @@ public T intercept(ClientInterceptor... interceptors) {
     return thisT();
   }
 
+  @Override
+  public T addTransportFilter(ClientTransportFilter transportFilter) {
```

Comment:
You can remove this one. ForwardingChannelBuilder extends ForwardingChannelBuilder2, so it'll be here already. We're set up to "never touch ForwardingChannelBuilder again."